ext/web/06_streams.js

```diff
@@ -5384,6 +5384,9 @@
      * @returns {void}
      */
     enqueue(chunk = undefined) {
+      if (chunk.byteLength === 0) {
+        return;
+      }
```

Comment:
This seems iffy, I think we might have to fix something in std/node instead, but let's see if WPT pass

Reply:
yeah i was pretty sure this isn't necessarily the right place but somehow node handles it

Reply:
Does Astro actually use Web streams instead of Node's "readable stream"?

Reply:
https://github.com/withastro/astro/blob/main/packages/astro/src/runtime/server/render/page.ts#L136

Reply:
I'm not too sure i kinda fumbled around with it and found this worked

Reply:
To be honest it feels like a bug in `ReadableStream` implementation in Node... With this change WPT seem to hang right now. Will debug it further

Reply:
or maybe Astro shouldn't be sending that in the first place I don't know

Reply:
That's a good point. @brenelz do you happen to know in which situations that happen in Astro?

Reply:
it seems to do it everytime even on the most basic hello world template

Reply:
this seems to throw when called with `chunk === undefined`

Reply:
So would doing this instead be a valid fix? `if (chunk && chunk.byteLength === 0) {`?

Reply:
Optional chaining would be a little cleaner. E.g. `chunk?.byteLength === 0`.

Reply:
seems like the wpt still hang :(

Reply:
Figured out it is this line causing the issue. It passes an empty Uint8Array. Is this valid?

https://github.com/web-platform-tests/wpt/blob/9f9c1d83e3296584cb301e6b560ddaa70abe1627/fetch/api/request/request-init-stream.any.js#L29

Reply:
So I think we can close these pr as this is the real fix
https://github.com/denoland/deno_std/pull/3062